inc/namespace.php

```diff
@@ -80,7 +80,23 @@ function merge_config_settings( array $config, array $overrides ) : array {
 			// Merge module settings together.
 			case 'modules':
 				foreach ( $value as $module => $settings ) {
-					$config[ $key ][ $module ] = array_merge( $config[ $key ][ $module ] ?? [], $settings );
+					// Check config settings syntax is valid.
+					$config_setting = $config[ $key ][ $module ];
+
+					if ( ! is_array( $config_setting ) && ! is_bool( $config_setting ) ) {
+						// phpcs:ignore
```

Comment:
Was this here because of local linter errors or hm-linter? We use this pattern all over the place so you shouldn't need it. Might need to check you have the latest HM coding standards installed.

If it is needed by hm-linter still I usually like to specify the sniff it's failing on eg `// phpcs:ignore Name.Of.Rule` so that it's only ignoring that rule.

Reply:
Good point, tbh I looked at how other code outputs the `trigger_error` and copied the approach.
Will keep in mind the rule specifics.